charts/incubator/hyperswitch-app/configs/misc.toml

```diff
@@ -1,174 +1,207 @@
+route_to_trace = {{ .Values.application.server.log.telemetry.route_to_trace }}
+bg_metrics_collection_interval_in_secs = {{ .Values.application.server.log.telemetry.bg_metrics_collection_interval_in_secs }}           # Interval for collecting the metrics in background thread
 
 [lock_settings]
-delay_between_retries_in_milliseconds = 500 # Delay between retries in milliseconds
-redis_lock_expiry_seconds = 180             # Seconds before the redis lock expires
+delay_between_retries_in_milliseconds = {{ .Values.application.server.lock_settings.delay_between_retries_in_milliseconds }} # Delay between retries in milliseconds
+redis_lock_expiry_seconds = {{ .Values.application.server.lock_settings.redis_lock_expiry_seconds }}             # Seconds before the redis lock expires
 
 [master_database]
-pool_size = "20"
-connection_timeout = 10   
-queue_strategy = "Fifo"
+pool_size = "{{ .Values.application.server.master_database.pool_size }}"
+connection_timeout = {{ .Values.application.server.master_database.connection_timeout }}   
+queue_strategy = "{{ .Values.application.server.master_database.queue_strategy }}"
 
 [generic_link]
+[generic_link.payment_method_collect]
+sdk_url = {{ include "hyperswitchWeb.hyperloaderUrl" . | quote }}
+expiry = 900
+
 [generic_link.payment_method_collect.ui_config]
-theme = "#4285F4"
-logo = "https://app.hyperswitch.io/HyperswitchFavicon.png"
-merchant_name = "HyperSwitch"
+theme = "{{ .Values.application.server.generic_link.payment_method_collect.ui_config.theme }}"
+logo = "{{ .Values.application.server.generic_link.payment_method_collect.ui_config.logo }}"
+merchant_name = "{{ .Values.application.server.generic_link.payment_method_collect.ui_config.merchant_name }}"
+
 [generic_link.payment_method_collect.enabled_payment_methods]
-card = "credit,debit"
-bank_transfer = "ach,bacs,sepa"
-wallet = "paypal,pix,venmo"
+card = "{{ .Values.application.server.generic_link.payment_method_collect.enabled_payment_methods.card }}"
+bank_transfer = "{{ .Values.application.server.generic_link.payment_method_collect.enabled_payment_methods.bank_transfer }}"
+wallet = "{{ .Values.application.server.generic_link.payment_method_collect.enabled_payment_methods.wallet }}"
+
+[generic_link.payout_link]
+sdk_url = {{ include "hyperswitchWeb.hyperloaderUrl" . | quote }}
+expiry = 900
 
 [generic_link.payout_link.ui_config]
-theme = "#4285F4"
-logo = "https://app.hyperswitch.io/HyperswitchFavicon.png"
-merchant_name = "HyperSwitch"
+theme = "{{ .Values.application.server.generic_link.payout_link.ui_config.theme }}"
+logo = "{{ .Values.application.server.generic_link.payout_link.ui_config.logo }}"
+merchant_name = "{{ .Values.application.server.generic_link.payout_link.ui_config.merchant_name }}"
+
 [generic_link.payout_link.enabled_payment_methods]
-card = "credit,debit"
+card = "{{ .Values.application.server.generic_link.payout_link.enabled_payment_methods.card }}"
```

Comment:
why is this need to be from values.yaml? isnt this a feature we provide?

Reply:
To generate readme and give control to users to easily edit it if needed